wrappers/README.md

```diff
@@ -30,8 +30,8 @@ If you downloaded a zip of the source, you need to go back to github to identify
 1. Download a zip using the GitHub download button in that tree, eg `realm-object-store-fb2ed6aa0073be4cb0cd059cae407744ee883b77.zip`
 1. Unpack its contents into `wrappers/src/object-store`
 
-Building Wrappers on macOS
---------------------------
+Building iOS and Android Wrappers on macOS
+------------------------------------------
```

Comment:
Then I guess https://github.com/realm/realm-dotnet/pull/1164/files/80880d8ec0b4d69f733a6a486a7044f3e9913528#diff-bebcb6e63d811c0b5f18fba9b7a7da3eR50 should be longer